src/interceptors/fetch/index.ts

```diff
@@ -22,16 +22,16 @@ export const interceptFetch: Interceptor = (observer, resolver) => {
   window.fetch = async (input, init) => {
     const ref = new Request(input, init)
```

Comment:
How about we call this `request`?

Reply:
Done in https://github.com/mswjs/interceptors/pull/186/commits/3dee65e9c153623812575c32bbc23af6342ddc87